specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/examples/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,52 @@
+{
+  "parameters": {
+    "api-version": "2020-03-20",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1",
+    "clusterName": "cluster1",
+    "cluster": {
+      "sku": {
+        "name": "AV20"
```

Comment:
Is it possible to change to "av20'? Because in response it is "av20" after testing.